src/hooks/NftCollectionMetadata.ts

```diff
@@ -0,0 +1,26 @@
+import { ipfsGetWithFallback } from 'lib/api/ipfs'
+import { NftCollectionMetadata } from 'models/nftRewardTier'
+import { useQuery } from 'react-query'
+import { cidFromUrl } from 'utils/ipfs'
+
+// gets Nft Collection metadata from IPFS
+export default function useNftCollectionMetadata(uri: string | undefined) {
```

Comment:
This isn't being used right now but will be necessary for changing NFT collection metadata in a follow-up